### PR TITLE
Improved functionality:

### PR DESCRIPTION
**Pull request by rjgoodin**
4 different "leaves"
Uses resistant cells
Renamed some variables.

### CODE REVIEW

File: models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+false
+"" ""
+PENS
+"infected" 1.0 0 -16777216 true "" "plot count cells with [infected?]"
+"resistant" 1.0 0 -7500403 true "" "plot count cells with [resistant?]"
+
+@#$#@#$#@
+## WHAT IS IT?
+
+(a general understanding of what the model is trying to show or explain)
```
Comment (alexanian) [inline models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo]: Should you guys fill this in?

**Comment by rjgoodin** [comment]
I just added some comments in there. Comments are good, right? 
Not super-descriptive, but general overview.

**Comment by jrhawley** [comment]
Yeah, comments look good. They're general enough and not overbearing, so that looks good to me

**Comment by alexanian** [comment]
I'm going to merge this, but then either @rjgoodin or @smile4life should try to unify the scripts into one, they have different names atm

**Pull request merged**